src/common/node/childProcess.ts

```diff
@@ -34,13 +34,19 @@ interface ISpawnOptions {
     detached?: boolean;
 }
 
+export interface IExecRejection {
```

Comment:
I'm implementing that suggestion in a different PR, so we can discuss it with the whole team